cnd/src/http_api/orderbook.rs

```diff
@@ -226,11 +226,11 @@ pub async fn get_orders(facade: Facade) -> Result<impl Reply, Rejection> {
 #[derive(Clone, Debug, Deserialize)]
 struct MakeOrderBody {
     position: Position,
+    rate: Rate,
```

Comment:
This is no good, please review @D4nte's work on floating point math (in project Nectar).

Reply:
I have looked at the code in project nectar. I don't understand why comit-rs requires the floating point conversion? I will add decimal string deserialisation but I still don't see why we need floating point conversions

Reply:
Even without the floating point comparison (assuming satoshi or wei integers) we still need to be able to multiply them together, this requires shifting decimal places to account for the conversion to/from sats and wei. This is what is being worked on in `float_math.rs`